### DISCUSSION TITLE
ScriptReference/AssetDatabase.DeleteAsset

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#254] [Notes]
# ScriptReference/AssetDatabase.DeleteAsset

https://docs.unity3d.com/ScriptReference/AssetDatabase.DeleteAsset.html

**Comment by radiatoryang** [comment]
Make sure you use [`DestroyImmediate()`](https://docs.unity3d.com/ScriptReference/Object.DestroyImmediate.html) if deleting a sub-asset (and then re-save or re-import the asset afterwards).
